`action/deps.js up-to-date` in CI frequently segfaults

We have a CI step where we check `action/deps.js` is up-to-date. This step recently has been very flaky - it fails with segmentation fault.

Example: https://github.com/denoland/deployctl/actions/runs/11465509277/job/31904102244#step:8:147

```
/home/runner/work/_temp/4a00770a-1094-47c0-a560-55297bec2a0b.sh: line 4:  2204 Segmentation fault      (core dumped) deno run --allow-read --allow-env --allow-net ./tools/bundle.ts ./src/utils/mod.ts > ./action/latest.deps.js
Error: Process completed with exit code 139.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -43,7 +43,9 @@ jobs:
         run: deno check deployctl.ts
 
       - name: action/deps.js up-to-date
-        if: matrix.deno == 'stable'
+        # On Linux the bundle command often segfaults. We use windows as a temporary
+        # workaround. See https://github.com/denoland/deployctl/issues/338
+        if: runner.os == 'Windows' && matrix.deno == 'stable'
         run: |
           # @deno/emit doesn't work if JSR modules are not in the cache.
           # This is a workaround to cache the JSR modules beforehand.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -43,7 +43,7 @@ jobs:
         run: deno check deployctl.ts
 
       - name: action/deps.js up-to-date
-        if: runner.os == 'Linux' && matrix.deno == 'stable'
+        if: matrix.deno == 'stable'
         run: |
           # @deno/emit doesn't work if JSR modules are not in the cache.
           # This is a workaround to cache the JSR modules beforehand.

